tv/tv.cpp

```diff
@@ -654,21 +658,10 @@ struct TVNewPass : public llvm::PassInfoMixin<TVNewPass> {
 
   void run(llvm::Module &M,
            const function<llvm::TargetLibraryInfo*(llvm::Function&)> &get_TLI) {
-    // Check TVFinalizePass::finalized again because TVNewPass can be
-    // called after TVFinalizePass registered by
-    // registerOptimizerLastEPCallback.
-    // It can happen when a default pipeline (O3, O2, ...) is used.
-    if (TVFinalizePass::finalized)
-      return;
```

Comment:
In the past, I found that this case never happened. I'll recheck whether it indeed is by running whole LLVM unit tests.